otel/starter/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/interceptor"
+
+	otelworkflow "github.com/temporalio/samples-go/otel"
```

Comment:
```suggestion
	"github.com/temporalio/samples-go/otel"
```

No need for package alias and doesn't seem purely about workflows

Reply:
it will now clash with "go.temporal.io/sdk/contrib/opentelemetry"